docs/source/interfaces/muon/muon_analysis_plotting_window.rst

```diff
@@ -19,13 +19,6 @@ It displays the selected data for a fit and if a fit is requested this is also s
 In the sequential fitting tab, the user can select the data to plot by highlighting the different rows in the table.
 
 
-Model Data
```

Comment:
why have we deleted this?

Reply:
The fit data window is only in Muon analysis so they are sitll different.

Reply:
ah yes good spot!